include/vrv/custos.h

```diff
@@ -50,6 +50,11 @@ class Custos : public LayerElement, public PitchInterface, public PositionInterf
      */
     bool IsSupportedChild(Object *object) override;
 
+    /**
+     * Return a SMuFL code for the custos
+     */
+    wchar_t GetCustosGlyph(const data_NOTATIONTYPE) const;
```

Comment:
For consistency add `notationType` parameter name

Reply:
sure, also added in accidentals and clef.